### ISSUE TITLE
Stepping the source causes an exception on Windows

### ISSUE THREAD

**Issue by dlsniper** [derekparker/delve#716]
1. What version of Delve are you using (`dlv version`)?
Version: 0.12.1
2. What version of Go are you using? (`go version`)?
go version go1.8rc3 windows/amd64
3. What operating system and processor architecture are you using?
Windows 10 Pro 14986.1001 /  Intel Core i7-6700HQ
4. What did you do?
```go
package main

import "fmt"

func main() {
	a := "value"
	fmt.Printf("%s\n", a)
}
```

```
go build -gcflags "-N -l" -o 3227.exe 3227.go
dlv.exe exec 3227.exe
b 3227.go:6
c
s
```
5. What did you expect to see?
```
value

```
6. What did you see instead?
```
Exception 0xc000001d 0x0 0x0 0x48a5c3
PC=0x48a5c3

main.main()
        D:/go/src/github.com/dlsniper/u/gog/3227/3227.go:7 +0x83
rax     0x498380
rbx     0x50be08
rcx     0xc042020000
rdi     0x49f1c0
rsi     0x1
rbp     0xc042031f78
rsp     0xc042031ed8
r8      0x338
r9      0x0
r10     0xc0420401c0
r11     0xc0420401c0
r12     0x1
r13     0x1d
r14     0x1c0
r15     0x200
rip     0x48a5c3
rflags  0x10216
cs      0x33
fs      0x53
gs      0x2b
Process 23216 has exited with status 2
```

**Comment by dlsniper** [comment]
Seems to be fixed in: https://github.com/derekparker/delve/pull/667
Let me know if I should close this or wait for that to be closed first.

**Comment by aarzilli** [comment]
Should be fixed by 8724b3fce73dac967aaec1460fbd0fe3c0e178df

**Comment by dlsniper** [comment]
I confirm it is, thank you!
